lib/services/systemd.c

```diff
@@ -586,25 +605,7 @@ systemd_unit_metadata(const char *name, int timeout)
         desc = crm_strdup_printf("Systemd unit file for %s", name);
     }
 
-    meta = crm_strdup_printf("<?xml version=\"1.0\"?>\n"
-                           "<!DOCTYPE resource-agent SYSTEM \"ra-api-1.dtd\">\n"
-                           "<resource-agent name=\"%s\" version=\"" PCMK_DEFAULT_AGENT_VERSION "\">\n"
-                           "  <version>1.0</version>\n"
-                           "  <longdesc lang=\"en\">\n"
-                           "    %s\n"
-                           "  </longdesc>\n"
-                           "  <shortdesc lang=\"en\">systemd unit file for %s</shortdesc>\n"
-                           "  <parameters>\n"
-                           "  </parameters>\n"
-                           "  <actions>\n"
-                           "    <action name=\"start\"   timeout=\"100\" />\n"
-                           "    <action name=\"stop\"    timeout=\"100\" />\n"
-                           "    <action name=\"status\"  timeout=\"100\" />\n"
-                           "    <action name=\"monitor\" timeout=\"100\" interval=\"60\"/>\n"
-                           "    <action name=\"meta-data\"  timeout=\"5\" />\n"
-                           "  </actions>\n"
-                           "  <special tag=\"systemd\">\n"
-                           "  </special>\n" "</resource-agent>\n", name, desc, name);
+    meta = crm_strdup_printf(METADATA_FORMAT, name, desc, name);
```

Comment:
sure that no high-level-tool or anything is using that special-tag? with 1.1 should even verify rng ...

Reply:
Good question. I suspect it was added here just because 1.0 technically marked special as required, even though in practice no agent used it. I suppose it's safer and doesn't hurt anything to add it back.